### DISCUSSION TITLE
Telegraf / Grafana

### DISCUSSION THREAD

**Discussion by Adminius** [hoylabs/OpenDTU-OnBattery#130] [Q&A]
Hi,

is there ready to use Telegraf + Grafana configs for "onBattery" version available?
I'm struggeling to du it by my self (never did something like before)

maybe one good example can be good included in to readme?

**Comment by helgeerbe** [comment]
I doubt that there is a ready to use config. 
First you might need an influxdb.
Depending what you are using, you could store OpenDTU Data via home homeassistant, node red, or telegraf in influxdb.
Grafana is just a tool to visualize the data in influxdb. But influxdb 2.x hat also visualizing capabilities.

By the way. Home Assistant comes with an energy dasboard, which is quite nice.

![image](https://user-images.githubusercontent.com/59169507/224022896-9491d806-32b2-469c-ab8b-7fedede84bcc.png)

**Reply by Adminius** [reply]
Hi, looks cool. But HA is a bit oversized, IMHO
This was my first attempt with TIG Stack:
![image](https://user-images.githubusercontent.com/17052262/224031194-b072ffbf-592a-479f-ad88-73994324d9a3.png)

Raspi with Mosquitto, Telegraf, InfluxDB, Grafana.
Maybe, it would be possible to use a HTTP API instead of MQTT

P.S. not tested: https://github.com/orcunbaslak/mqtt-telegraf-influxdb-grafana
